src/java/fr/paris/lutece/plugins/workflow/modules/appointmentants/service/TaskAntsAppointmentService.java

```diff
@@ -451,6 +471,32 @@ public static Map<String, String> getAppointmentData( HttpServletRequest request
 		return appointmentDataMap;
 	}
 
+	/**
+	 * Get the AppointmentDTO containing the previous data of an appointment. It is retrieved
+	 * from the request's attributes
+	 * 
+	 * @param request
+	 * 				The request containing the AppointmentDTO in its attributes
+	 * @return
+	 * 				The AppointmentDTO Object containing the previous data if it was found,
+	 * 				returns null otherwise
+	 */
+	private static AppointmentDTO getOldAppointment( HttpServletRequest request )
+	{
+		AppointmentDTO oldAppointment = null;
+
+		try
+		{
+			// Retrieve the previous appointment from the request's parameters
+			oldAppointment = ( AppointmentDTO ) request.getAttribute( AppointmentUtilities.OLD_APPOINTMENT_DTO );
+		}
+		catch ( Exception e )
+		{
+			AppLogService.error( BEAN_SERVICE, e );
```

Comment:
Retire la journalisation de l'erreur dans les journaux. Sinon, chaque fois que nous supprimons un rendez-vous dans Ants et que cette suppression n'est pas liée à un report, une trace de l'exception apparaîtra dans les journaux. Cela pourrait nous égarer en cas de problème.
Tu peux la remplacer par un message d'info AppLogService.info(  "remove appointment ants: {}" , e.getMessage());

Reply:
Thanks, the message level has been update